src/components/utils/DeviceUtils.tsx

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
-import { Hidden } from '@material-ui/core';
+import { Theme, useMediaQuery } from '@material-ui/core';
 
 interface IProps {
-  children: React.ReactChild | React.ReactChild[];
+  children: JSX.Element | JSX.Element[];
 }
 
-export const Mobile = ({ children }: IProps): JSX.Element => <Hidden smUp>{children}</Hidden>;
```

Comment:
What's wrong with `<Hidden>`?

Reply:
`<Hidden>` was [deprecated](https://mui.com/material-ui/react-hidden/) in favor `useMediaQuery`. Also the API for it has changed so it wasn't even compiling anyway.

Reply:
Ahh ok. Seems like we could still make a small helper function for all of the components below. Also, why the `.map(child => child)` for the array case?

Reply:
Must've been something weird with the typechecker, but looks like no need for it anymore. Also made that helper.